src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -1197,6 +1199,15 @@ void Terminal::_AdjustCursorPosition(const til::point proposedPosition)
                         _selection.reset();
                     }
                 }
+
+                // If we still have a selection, make sure to sync the pivot
+                // with whichever value is the right one.
+                //
+                // Failure to do this might lead to GH #14462

```

Comment:
```suggestion
```

I respect that this is true, but also... `git blame` could probably tell you that, and I don't know that every time we make a change we need to decorate the code with signposts that say "this was a bug back when xyz" :)

Reply:
sorry missed this - FWIW I still add these out of habit. After being through 4 different source control trees, I find it's 1000% easier to leave the history for weird lines of code next to them in the source. Then, when one of these lines is inevitably refactored, we can see the comment moved/deleted and check the original bug for regressions, rather than manually `git blame`ing every time we _think_ a line of code comes from a weird bug report. 

That's my 2c. LOC are cheap.